tests/test_helpers.py

```diff
@@ -178,3 +183,15 @@ def get_parameters_from_parameter_dict(parameter_dict):
         api_data_types,
         api_location_types,
     )
+
+
+def test_validate_parameter_format_for_mastr_init(db):
+    engine_list_working = ["sqlite", "docker-postgres", db._engine]
+    engine_list_failing = ["HI", 12]
+
+    for engine in engine_list_working:
+        assert validate_parameter_format_for_mastr_init(engine) is None
+
+    for engine in engine_list_failing:
+        with pytest.raises(ValueError):
+            validate_parameter_format_for_mastr_init(engine)
```

Comment:
Is this implementation resulting in a test failure for all other values apart from `["sqlite", "docker-postgres", db._engine]` and not only a failure for `["HI", 12]` ?

Reply:
Yes, but it is hard to test for such a large amount of failing values, so I just chose an exemplary string and integer value that makes the code fail.